qiita_pet/support_files/doc/source/tutorials/portals.rst

```diff
@@ -0,0 +1,38 @@
+.. _portals:
+
+.. index:: portals
+
+User Portals
+============
+
+User portals allow a user to cordon off specific studies and analyses into their own user interface, which we call a portal, selectively slicing the data available through that portal.
+
+The main `QIITA` portal gives access to all studies in the database, while adding studies to a specific portal and changing the config file to point to the portal will show the same interface, but with only the selected studies. To change the portal being accessed, simply change the `portal` setting in the qiita config file. This must correspond to a portal name in the database.
```

Comment:
Thanks, aaahhh, got it, then it might make sense to add the extra
backtick or remove the backticks altogether.

On (Jul-04-15|14:59), Joshua Shorenstein wrote:
> > @@ -0,0 +1,38 @@
> > +.. _portals:
> > +
> > +.. index:: portals
> > +
> > +User Portals
> > +============
> > +
> > +User portals allow a user to cordon off specific studies and analyses into their own user interface, which we call a portal, selectively slicing the data available through that portal.
> > +
> > +The main `QIITA` portal gives access to all studies in the database, while adding studies to a specific portal and changing the config file to point to the portal will show the same interface, but with only the selected studies. To change the portal being accessed, simply change the `portal` setting in the qiita config file. This must correspond to a portal name in the database.
> 
> Redone for 80 columns. The capital qiita is required, as that's what is in the system. If you use `Qiita` it will fail.
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/biocore/qiita/pull/1314/files#r33890273